python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_deflection_tube.py

```diff
@@ -146,19 +265,107 @@
+# calibration.
+# In order to clearly see these results, the next table
+# presents the true standard deviation of the
+# output error of observation compared to the hypothesis used
+# in Gaussian calibration.
+# In a practical study, the true sigma is unknown.
+#
+# +-----------------+-----------------+-------------------+
+# | Output          | Sigma True      | Sigma hypothesis  |
+# +=================+=================+===================+
+# | Deflection      | 0.1e-6          | 0.2e-6            |
+# +-----------------+-----------------+-------------------+
+# | Left angle      | 0.5e-6          | 0.3e-6            |
+# +-----------------+-----------------+-------------------+
+# | Right angle     | 0.5e-6          | 0.3e-6            |
+# +-----------------+-----------------+-------------------+
+#
+# **Table 3.** For each output, the true standard deviation of the
+# output error of observation compared to the hypothesis used
+# in Gaussian calibration.
+#
+# One of the hypotheses of the Gaussian calibration is
+# that the observation errors are Gaussian.
+# In order to check this hypothesis, we plot the distribution
+# of the residuals before and after calibration.
+# Furthermore, we plot the distribution of the observation
+# errors, which is an hypothesis of the Gaussian calibration.
+# Since there are three output, there are three residuals to check.
 graph = calibrationResult.drawResiduals()
-view = viewer.View(graph)
+view = viewer.View(
+    graph,
+    figure_kw={"figsize": (13.0, 4.0)},
+    legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+)
+plt.subplots_adjust(wspace=0.3, left=0.05, right=0.8)
 
 # %%
+# We see that the distribution of the residuals after calibration
+# is centered on zero: this is a good point, since this shows that
+# we have roughly equal chances to over or under predict the output
+# when we use the calibrated model.
+# We see that the distribution of each the residuals after calibration
+# is relatively close to the Gaussian assumption that we used.
+# For the first output, i.e. the deflection, the calibrated distribution
+# of the residuals has a distribution which has spread lower
+# than the hypothesis.
+# This corresponds to the parameters that we used, since the true
+# standard deviation of the errors is equal to 0.1e-6 while
+# we used the 0.2e-6 hypothesis.
+# The opposite happens for the left and right angle residuals:
+# the calibrated residuals are narrower than the normal hypothesis
+# that we used.
+# This is because we used 0.3e-6 as the hypothesis while the
+# true value is 0.05e-5.
+
+# %%
+# Finally, we observe the prior and posterior distribution of each
+# parameter.
 graph = calibrationResult.drawParameterDistributions()
-view = viewer.View(graph)
+view = viewer.View(
+    graph,
+    figure_kw={"figsize": (10.0, 4.0)},
+    legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+)
+plt.subplots_adjust(wspace=0.3, left=0.05, right=0.8)
 
 plt.show()
```

Comment:
```suggestion
```
Unnecessary, I think.

Reply:
I thought like you before, but then realized that a Python script that does not contain that line does not create the graphics windows that contain the Matplotlib graphics. This is why I suggest to leave it as is. Do you agree?

Reply:
I don't get it. Other examples don't use it and are displayed fine: https://output.circle-artifacts.com/output/job/bf84824d-8f54-46c4-9853-7258c015a60b/artifacts/0/html/auto_calibration/bayesian_calibration/plot_ackley_distribution.html#sphx-glr-auto-calibration-bayesian-calibration-plot-ackley-distribution-py

Reply:
I agree with that, but that is not the point. In a GNU/Linux terminal, consider the command line:
```bash
$ python3 python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_chaboche.py
```
No plot appear. At that point, we may wonder if there are any graphics command in the script, because it would not make any visible difference.  If, on the other hand, the `plt.show()` command is executed at the bottom of the script, then all graphics windows suddenly appear (each plot has its own graphics window, with zoom in/out).

So removing `plt.show()` is a trade-off:
- if the goal is to produce the doc of OpenTURNS, then we should remove it,
- if the goal is to produce Python scripts that open the graphics windows, then we should write it.

I can remove it if you will, but I thought that you might want to consider that before requiring to remove it. I agree, however, that there are inconsistencies in the doc, since some examples have that, while other do not.

Reply:
I did not consider that because I rarely run scripts (especially scripts designed to produce plots) that way: IDEs are more comfortable.

Doesn't View.ShowAll have the same effect as plt.show? If so, then using it would allow us not to import matplotlib just for the sake of this display.